src/lib/ext-activitylog.js

```diff
@@ -244,6 +244,7 @@ class View {
 
   clearTable() {
     this.logView.clearTable();
+    this.logView.closeDetailSidebar();
```

Comment:
This is still the same change that was in #86 (just the unrelated empty line change has been removed from what was in #86), please reread the previous feedback comments and update this PR accordingly.

Reply:
Okay sir, I will make changes in log-view-element.js according to you and thanks

Reply:
sir, this pr is not eligible?

Reply:
yes, I described the reason why we should be fix the issue in another source file (in `log-view-element.js` instead of `ext-activitylog.js`) in my previous feedback comments, eg. as I did mention in the issue here https://github.com/mozilla/extension-activity-monitor/issues/56#issuecomment-704432760:
> The log details sidebar is part (and managed by) the webcomponent defined in log-view-element.js.

In other words: given that there shouldn't be any real scenario that would require to clear the table but keep the sidebar open, we can assign the responsibility to close the sidebar automatically to the log-view webcomponent itself and make it something that happens internally in the `log-view` webcomponent when `logView.clearTable()` has been called.

have you tried to think / look into how the same change you applied here could be moved in `log-view-element.js`?

As you can see from this diff, both `clearTable` and `closeDetailSidebar` are methods of the `log-view` webcomponent defined in `log-view-element.js`.

From inside the [`clearTable` method defined in `log-view-lement.js`](https://github.com/mozilla/extension-activity-monitor/blob/a981ef40ccf2fa41fdc68fcddd0af771a1b43297/src/lib/web-component/log-view/log-view-element.js#L136-L139) we can call `this.closeDetailsSidebar()` and avoid any change to ext-activitylog.js.

(as a side effect of this choice, this behavior would be provided by the log-view component itself and so it would be supported even if we do reuse the log-view webcomponent into some other extension page that it isn't managed by the MVC implemented in ext-activitylog.js).